dxp/resource_injection.md

```diff
@@ -0,0 +1,106 @@
+# DXP Support for Injection of Dynamic Resources
+
+This document describes different APIs that can be used with DXP to inject resources (`.js`, `.css`, ...) to a HTML page, thus letting the user extend currently existing artifacts without any need to overwrite them (like it was previously done with hooks in older Liferay versions).
```

Comment:
"a HTML" → "an HTML" ([reasons](https://english.stackexchange.com/questions/25742))

Reply:
:-o I used to apply that rule, but then saw some place that some people stick to the letter, not the sound.

[RAE](https://www.rae.es/) for English now!

![protest](https://media.giphy.com/media/3ohjVabojNBmsqeqg8/giphy.gif)